core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -454,6 +455,16 @@ public ClientFactory build() {
                                                                maxNumEventLoopsFunctions);
         }
 
+        // FIXME(ikhoon) This workaround enables DefaultAddressResolverGroup by `useDefaultDnsResolver` flag.
+        // The DefaultAddressResolverGroup which calls blocking operation uses JDK's built-in domain name lookup
+        // mechanism. However DNS result is easy to cache, so it doesn't have a big impact on performance.
+        // The code that uses DefaultAddressResolverGroup could be removed after resolving the following issue.
+        // https://github.com/line/armeria/issues/2243
```

Comment:
- The first sentence of a FIXME/TODO should explicitly state 'what has to be done'.
- Please indent properly if this whole paragraphs is part of the FIXME:
  ```
  // FIXME(ikhoon): blah
  //                blaah
  //                blaaah
  ```
- Missing colon after FIXME